code/game/machinery/bioprinter.dm

```diff
@@ -69,7 +70,8 @@
 		stored_matter -= products[choice][2]
 		var/new_organ = products[choice][1]
 		var/obj/item/organ/O = new new_organ(get_turf(src))
-		O.is_printed = TRUE // Heist stuff.
+		if(choice == !"tooth")
```

Comment:
The code is assuming every thing printed is an organ, which is why you need this hack. Rewrite it so it checks whether the printed thing is an organ before assuming it's one.